tools/cmake_test.py

```diff
@@ -265,8 +265,21 @@ def _run(self):
                      args=self.test_args)
         logger.info(cmd)
 
+        # setup llvm symbolizer. first look for location in ci, then in redpanda
+        # vbuild directory. if none, then asan will look in PATH
+        env = os.environ.copy()
+        llvm_symbolizer = shutil.which("llvm-symbolizer", path="/vectorized/llvm/bin")
```

Comment:
`/vectorized/llvm/bin` is the location in CI?

Reply:
Correct. Here is another location where this knowledge is baked in (for better or worse)

https://github.com/redpanda-data/redpanda/blob/dev/tools/cmake_test.py#L127-L131

```
        # CI: use prior knowledge of location where tool is
        # copied in Dockerfile for build image
        ci_location = "/vectorized/bin/seastar-addr2line"
        if os.path.exists(ci_location):
            return ci_location
```